umbraco/Umbraco.CMS.Backoffice
# Feature: Select all variants option in publishing dialogs

## Description

This PR adds a "Select all" option to the "publishing" dialogs
* Save
* Save and Publish
* Schedule
* etc.

What to test:
* Make sure that when selecting all, all allowed variants are selected
* Make sure only the allowed languages are chosen

<img width="498" alt="Screenshot 2024-10-04 at 13 03 03" src="https://github.com/user-attachments/assets/b71a6e93-068d-4ffb-9173-6eb43566e496">

 
## Types of changes
